data/json/items/armor/ballistic_armor.json

```
@@ -642,6 +625,149 @@
     "looks_like": "esbi_plate",
     "material": [ "damaged_ceramic" ]
   },
+  {
+    "id": "ballistic_vest_heavy",
+    "type": "ARMOR",
+    "category": "armor",
+    "//": "Based on US Army IOTV. Weight from size M base piece plus lower back protector, data from: https://www.moore.army.mil/tenant/lrc/content/pdf/IOTV%20GEN%20III%20TM%2010-8470-210-10.pdf ",
+    "name": { "str": "heavy ballistic vest" },
```

```suggestion
    "name": { "str": "US heavy ballistic vest" },
```
I think this would work better to express that it is the IOTV.